onnx/version_converter/adapters/axes_attribute_to_input.h

```diff
@@ -16,27 +17,28 @@ class AxesAttributeToInput : public Adapter {
   explicit AxesAttributeToInput(const std::string& op_name, const OpSetID& initial, const OpSetID& target)
       : Adapter(op_name, initial, target) {}
 
-  void attrToInput(std::shared_ptr<Graph> graph, Node* node, std::vector<int64_t> axes) const {
-    Tensor t;
-    t.elem_type() = TensorProto_DataType_INT64;
-    t.sizes() = std::vector<int64_t>{static_cast<int64_t>(axes.size())};
-    auto& data = t.int64s();
-    for (auto a : axes) {
-      data.emplace_back(a);
-    }
-
-    Node* constant = graph->create(kConstant);
-    constant->insertBefore(node);
-    constant->t_(kvalue, t);
-    node->addInput(constant->output());
+  void attrToInput(GraphProto* graph_proto, NodeProto* node_proto, std::vector<int64_t>& axes) const {
+    node_proto->attribute();
+    const std::string constant_node_name("aaa");
```

Comment:
Is this `"aaa"` thing intended? If so a comment may help

Reply:
I will clean up the code when it is clear that the approach is working. Sorry for the confusion.